addons/website/static/src/scss/website.scss

```diff
@@ -1278,6 +1278,95 @@ $-o-carousel-controllers-size: map-get($spacers, 5);
     }
 }
 
+// Single scroll animation for the carousel
+.o_slide_single {
+    --o_flex_value: 50%;
+    /* Set up css variables
+    (Define 1 to 4 cards in the carousel) */
+    .o_with_1_cards {
+        --o_flex_value: 100%;
+    }
+
+    .o_with_2_cards {
+        --o_flex_value: 50%;
+    }
+
+    .o_with_3_cards {
+        --o_flex_value: 33.33%;
+    }
+
+    .o_with_4_cards {
+        --o_flex_value: 25%;
+    }
+
+    @media (min-width: 768px) {
+        .carousel-inner {
+
+            // DISPLAY MULTIPLE CARDS IN A ROW
+
+            display: flex;
+
+            .carousel-item {
+                transition: transform 0.3s ease-in-out;
+            }
+
+            //TODO: Find why this is not working
+            //.carousel-item.active,
+            //.carousel-item.active ~ .carousel-item {
+            //    display: block;
+            //    flex: 0 0 var(--o_flex_value);
+            //    margin-right: 0;
+            //}
+
+            .carousel-item.active,
+            .carousel-item.active + .carousel-item,
+            .carousel-item.active + .carousel-item + .carousel-item,
+            .carousel-item.active + .carousel-item + .carousel-item + .carousel-item,
+            .carousel-item.active + .carousel-item + .carousel-item + .carousel-item + .carousel-item {
```

Comment:
it's not possible to do ?
```css
.carousel-item.active,
.carousel-item.active ~ .carousel-item {
```

Reply:
It does not work, I don't understand why since it's supposed to do the same thing. 
For all other modifications this notation works but for that particular rule, it does not.